java/code/src/com/redhat/rhn/domain/contentmgmt/ProjectSource.java

```diff
@@ -146,6 +146,20 @@ public ProjectSource(ContentProject project) {
         this.contentProject = project;
     }
 
+    /**
+     * Gets the source Label.
+     *
+     * @return source label
+     */
+    public abstract String sourceLabel();
```

Comment:
According to default naming, this should be `getSourceLabel`, no?

Reply:
well I tried... but this class is a jpa entity and with that, I get errors from hibernate that somehow tries to serialize all the getters "getter that is not a field". Any suggestion to avoid this? I don't feel like creating a wrapper class just for this

Reply:
this is no longer valid with my last commit